dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -0,0 +1,115 @@
+package com.dotcms.rest.api.v1.container;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotcms.repackage.javax.ws.rs.*;
+import com.dotcms.repackage.javax.ws.rs.core.Context;
+import com.dotcms.repackage.javax.ws.rs.core.MediaType;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.rest.InitDataObject;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.api.v1.site.SiteHelper;
+import com.dotcms.rest.exception.mapper.ExceptionMapperUtil;
+import com.dotcms.util.I18NUtil;
+import com.dotcms.util.PaginationUtil;
+import com.dotcms.util.pagination.ContainerPaginator;
+import com.dotcms.util.pagination.OrderDirection;
+import com.dotcms.util.pagination.SitePaginator;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.dotmarketing.util.WebKeys;
+import com.liferay.portal.model.User;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.io.Serializable;
+import java.util.Map;
+import static com.dotcms.util.CollectionsUtils.map;
+
+/**
+ * This resource provides all the different end-points associated to information
+ * and actions that the front-end can perform on the {@link com.dotmarketing.portlets.containers.model.Container}.
+ *
+ */
+@Path("/v1/containers")
+public class ContainerResource implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private final PaginationUtil paginationUtil;
+    private final WebResource webResource;
+
+    public ContainerResource() {
+        this(new WebResource(),
+                new PaginationUtil(new ContainerPaginator()));
+    }
+
+    @VisibleForTesting
+    public ContainerResource(final WebResource webResource,
+                             final PaginationUtil paginationUtil) {
+        this.webResource = webResource;
+        this.paginationUtil = paginationUtil;
+    }
+
+    /**
+     * Return a list of {@link com.dotmarketing.portlets.containers.model.Container}, entity response syntax:.
+     *
+     * <code>
+     *  {
+     *      contentTypes: array of Container
+     *      total: total number of Containers
+     *  }
+     * <code/>
+     *
+     * Url sintax: api/v1/container?filter=filter-string&page=page-number&per_page=per-page&ordeby=order-field-name&direction=order-direction&host=host-id
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 154 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)